tests/apps.py

```diff
@@ -2,5 +2,5 @@
 
 
 class TestsConfig(AppConfig):
-    name = 'Tests'
+    name = 'tests'
```

Comment:
without this change, i wasn't able to run tests! i would get the following error:

```
Traceback (most recent call last):
  File "/Users/tao/dev/django-service-objects/.venv/lib/python3.10/site-packages/django/apps/config.py", line 210, in create
    app_module = import_module(app_name)
  File "/Users/tao/.asdf/installs/python/3.10.5/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'Tests'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/tao/dev/django-service-objects/runtests.py", line 16, in <module>
    django.setup()
  File "/Users/tao/dev/django-service-objects/.venv/lib/python3.10/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/tao/dev/django-service-objects/.venv/lib/python3.10/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/Users/tao/dev/django-service-objects/.venv/lib/python3.10/site-packages/django/apps/config.py", line 212, in create
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Cannot import 'Tests'. Check that 'tests.apps.TestsConfig.name' is correct.
```